### PR TITLE
hg-status: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/hg-status.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Display only deleted (but tracked) files:
+
+`hg status --deleted`
+
+- Display changes in the working directory compared to a specified changeset:
+
+`hg status --rev {{revision}}`
+
+- Display only files matching a specified pattern:
```
Comment (sbrl) [inline pages/common/hg-status.md]: Is this a glob or a regex?

Comment (mfrw) [reply pages/common/hg-status.md]: Its a glob e.g `*.md`

Comment (pxgamer) [reply pages/common/hg-status.md]: Yeah, I think I mention that `hg` commands use globs in another `hg` command page. :thinking:

Comment (mebeim) [inline pages/common/hg-status.md]:
```suggestion
- Display only files matching a specified glob pattern:
```
Would it be clearer this way? Looks good both ways to me, just making sure.

Comment (pxgamer) [reply pages/common/hg-status.md]: Yeah, I guess. 👍

**Pull request merged**